### PR TITLE
Prevent continuous textDocument/hover error when hover is unsupported

### PR DESCRIPTION
**Pull request by junkblocker**

### CODE REVIEW

File: lua/docs-view.lua [hunk 1]
```diff
@@ -44,6 +44,18 @@ local function toggle()
...
+          local gotHover = false
+          for i = 1, #clients do
+            if clients[i].supports_method("textDocument/hover") then
+              gotHover = true
+              break
+            end
+          end
+          if not gotHover then
+            return
+          end
```
Comment (amrbashir) [inline lua/docs-view.lua]: could you move this logic one line above? just so `l, c` are defined next to where they are used

File: lua/docs-view.lua [hunk 2]
```diff
@@ -44,6 +44,18 @@ local function toggle()
       callback = function()
         if win and vim.api.nvim_win_is_valid(win) then
           local l, c = unpack(vim.api.nvim_win_get_cursor(0))
+          local clients = vim.lsp.buf_get_clients(0)
```
Comment (amrbashir) [inline lua/docs-view.lua]: Seems like `vim.lsp.buf_get_clients` is deprecated now and we should use `vim.lsp.get_active_clients`

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you for this awesome contribution. I sent a couple of nit-picks your way.

File: lua/docs-view.lua (hunk 2 above)
Comment (junkblocker) [reply lua/docs-view.lua]: I saw that but are you ok with requiring 0.8.0? I personally am. I'll make the change.

File: lua/docs-view.lua (hunk 1 above)
Comment (junkblocker) [reply lua/docs-view.lua]: done.

File: lua/docs-view.lua (hunk 2 above)
Comment (amrbashir) [reply lua/docs-view.lua]: I tried to see when was this API introduced but I couldn't find it.

I think this plugin needs to at least support 0.7 so I guess we could check nvim version and use `vim.lsp.get_active_clients()` on 0.8 and higher

Comment (junkblocker) [reply lua/docs-view.lua]: Done.

**Review by amrbashir — APPROVED** [review]
Thank you

**Pull request merged**